mne/decoding/transformer.py

```diff
@@ -170,7 +170,7 @@ def transform(self, epochs_data, y=None):
         memory usage may be large with big data.
         """
         return _sklearn_reshape_apply(self._scaler.transform, True,
-                                      epochs_data, y)
+                                      epochs_data)
```

Comment:
Two things
1. self.transform should not take `y`, [this is the deprecation sklearn generated](https://github.com/scikit-learn/scikit-learn/blob/f9332d56/sklearn/preprocessing/data.py#L659).
2. self.fit should have a default `y=None`